articles.tex

```diff
@@ -97,11 +97,10 @@
 A quorum of fifty percent of the Total Number of Possible Votes is required for passage.
 
 \asubsection{Semantic Changes}
-Any semantic change to the Constitution requires the change to be proposed in writing for discussion at a House Meeting.
-Any modifications made due to the discussion are added to the written proposal and the modified proposal is posted in the House during the week.
-The final proposal is presented the following week at the end of House Meeting.
-Attendance for the discussion on the change is not required, but only members who stay until the end of discussion will receive ballots.
-At the end of discussion, ballots are distributed for a ballot House vote as described in \ref{Balloted Vote}.
+Any semantic change to the Constitution requires the change to be brought up at House Meeting and proposed in writing for discussion at the following House Meeting.
+The text of the change may be modified up until the aforementioned House Meeting. 
+At the beginning of the discussion period, ballots are distributed for a balloted House vote as described in \ref{Balloted Vote}.
+Attendance for the discussion on the change is not mandatory.
```

Comment:
Thanks for the input, let me change a few things and see if that addresses most of your concern.